### PR TITLE
Add wiki tutorial on how to slim down the ram usage

### PR DESCRIPTION
**Pull request by 9SMTM6**
This is a suggestion from https://github.com/open-webui/open-webui/discussions/2583.

### CODE REVIEW

File: docs/tutorial/slim_down.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+A lot of the memory consumption is because of loaded ML models. Even if you use an external language model (OpenAI or un-bundled ollama) a lot of models may be loaded for additional purposes.
+
+As of v0.3.10 this includes:
+* Speach-to-text (defaults to whisper)
+* RAG Embedding engine (defaults to local SentenceTransformers model)
+* Image generation engine (disabled by default)
+
+The first 2 are enabled and set to local models by default. You can change the models in the admin planel (RAG: Documents category, set it to ollama or OpenAI, Speach-to-text: Audio section, OpenAI or WebAPI work).
+If you deploy via docker you can also set these with the following environment variables: `RAG_EMBEDDING_ENGINE: ollama`, `AUDIO_STT_ENGINE: openai`.
```
Comment (justinh-rahb) [inline docs/tutorial/slim_down.md:26]: Would note that this only works on _first_ deployment, after the container starts the values that are saved to `config.json` will take precedence.

**Comment by justinh-rahb** [comment]
I appreciate this addition to the docs and want to see it merged, but I think it can use some additional work to maintain consistency with the rest of the Docs site style.

File: docs/tutorial/slim_down.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+## TLDR
+
+Set the following environment variables: `RAG_EMBEDDING_ENGINE: ollama`, `AUDIO_STT_ENGINE: openai`.
+
+## Longer explanation
+
+A lot of the memory consumption is because of loaded ML models. Even if you use an external language model (OpenAI or un-bundled ollama) a lot of models may be loaded for additional purposes.
+
+As of v0.3.10 this includes:
+* Speach-to-text (defaults to whisper)
```
[the remark this answers is not in the document]
Comment (9SMTM6) [reply docs/tutorial/slim_down.md]: I put all text through deepl write to avoid wrong grammar.

File: docs/tutorial/slim_down.md (hunk 1 above)
Comment (9SMTM6) [reply docs/tutorial/slim_down.md]: I suspected that, this is how most images behave, if environment variables are not the single source of truth. But I wasn't sure of it, and didn't want to complicate things. But you're right, we should put this in there, and I will.

**Comment by 9SMTM6** [comment]
@justinh-rahb I'm not  sure that you mean precisely. I did actually look a bit at the other pages, but I wasn't able to find some real consistency there. I mean, [one of these pages](https://docs.openwebui.com/tutorial/apache) has a thank-saying in the footer. There didn't seem a consistent style.

If you've got more concrete points, please mention them. But the only things where I'd say its stylistically different from the docs were conscious choices, and I see no reason to change them:
* I mention the version and environment since the information might become out of date and might not lead to exactly the same result in other environments
* its pretty terse, but I see no point in unnecessary prose. 

If there's more things missing, there is a nice 'edit me' bottom at the end of the wiki where people can add  more context. Without the  information in this Wiki I was seriously considering giving up on having OpenWebUI, with it I'm a happy user. So I think its already very helpful in this state.

File: docs/tutorial/slim_down.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (justinh-rahb) [reply docs/tutorial/slim_down.md]: It's still spelt incorrectly.

**Pull request merged**